src/api/revurderingApi.ts

```diff
@@ -67,12 +69,33 @@ export async function beregnOgSimuler(
     });
 }
 
+export enum Revurderingshandling {
+    SendTilAttestering = 'SEND_TIL_ATTESTERING',
+    Forhåndsvarsle = 'FORHÅNDSVARSLE',
+}
+
+export async function forhåndsvarsleEllerSendTilAttestering(
+    sakId: string,
+    revurderingId: string,
+    revurderingshandling: Revurderingshandling,
+    fritekstTilBrev: string
+): Promise<ApiClientResult<SimulertRevurdering, RevurderingErrorCodes>> {
+    return apiClient({
+        url: `/saker/${sakId}/revurderinger/${revurderingId}/forhandsvarsleEllerSendTilAttestering`,
+        method: 'POST',
+        body: {
+            revurderingshandling,
+            fritekst: fritekstTilBrev,
+        },
+    });
+}
+
 export async function sendTilAttestering(
     sakId: string,
     revurderingId: string,
     fritekstTilBrev: string,
     skalFøreTilBrevutsending?: boolean
-): Promise<ApiClientResult<RevurderingTilAttestering>> {
+): Promise<ApiClientResult<RevurderingTilAttestering, RevurderingErrorCodes>> {
```

Comment:
Hva er forskjellen på å bruke komma versus pipe her?

Reply:
Usikker på om jeg forstår hvorfor det skal pipes her. ApiClientResult<Success, Error> bare forteller oss om typene som vi får tilbake hvis kallet var OK, eller en error.

Reply:
jeg bare sammenlignet den med 138: `): Promise<ApiClientResult<SimulertRevurdering | RevurderingTilAttestering>> {`

Reply:
Men det betyr bare at andreparameter sikkert er default undefined

Reply:
@ingara kan sikkert dobbelt-sjekke at jeg sier dette riktig:
`): Promise<ApiClientResult<SimulertRevurdering | RevurderingTilAttestering>> {` Der er begge revurderingene et OK-case. Valget man tar etter forhåndsvarsling, er å sende til attestering, eller endre på fradragene.  

I tilfellet vi velger å endre på fradragene, så lagrer bare backend at vi har gjort et valg, og gir oss tilbake revurderingen. Denne er da fortsatt simulert.

Andre caset er at vi fortsetter med de samme inputene, men sender til attestering. 

Success-caset våres trenger en type. Mens error, har en default, så vi ikke trenger å alltid fylle den ut. 

RevurderingErrorCodes er bare en felles type vi lagde for å samle alle errors backend kan gi oss for revurdering som vi kan stappe inn i RevurderingsKallFeilet komponenten.